### PR TITLE
Tabs styling issue

### PR DESCRIPTION
**Pull request by agul89**
Modified the structure and styling to be more flexible and adjustable.
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: report/src/main/webapp/assets/sass/_toolbar.scss [hunk 1]
```diff
@@ -55,9 +50,8 @@
   }
 
   &-link {
+    width: calc(100% - 200px);
```
Comment (mateuszluczak) [inline report/src/main/webapp/assets/sass/_toolbar.scss]: `@include` at the top of the rest properties

File: report/src/main/webapp/assets/sass/_toolbar.scss [hunk 2]
```diff
@@ -68,24 +62,26 @@
...
+      border-right: solid 1px $border_color;
     }
   }
 
   &-blocks {
-    float: right;
-
+    display: flex;
+    flex-wrap: nowrap;
     &.is-absolute {
```
Comment (mateuszluczak) [inline report/src/main/webapp/assets/sass/_toolbar.scss]: New line above as it was before - follow the convention

File: report/src/main/webapp/app/layout/main/test/mainView.test.view.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
     limitations under the License.
 
 -->
-<div class="main page-main">
+<div class="page-main">
```
Comment (mateuszluczak) [inline report/src/main/webapp/app/layout/main/test/mainView.test.view.html]: Wasn't `main` class responsible for handling expanding and collapsing sidebar?
Can we safely remove it without losing that functionality? Is `main` class used anywhere now?

Comment (agul89) [inline report/src/main/webapp/app/layout/main/test/mainView.test.view.html]: 'main' is also used with 'toolbar-holder', so expanding and collapsing works properly

File: report/src/main/webapp/assets/sass/_report.scss [hunk 4]
```diff
@@ -309,19 +309,6 @@ ul {
     }
   }
 }
-.bg-shield {
```
Comment (wiiitek) [inline report/src/main/webapp/assets/sass/_report.scss]: Thank you for cleaning this up! 👍

**Pull request merged**